### PR TITLE
Update polly to use port 7978 for the admin interface

### PR DESCRIPTION
**Pull request by dvonthenen**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,4 +23,4 @@ RUN cp /go/bin/polly /usr/local/ && chmod u+x /usr/local/polly
 # Start the Polly Service
 ENTRYPOINT polly service start -f
 
-EXPOSE 7979 7980
\ No newline at end of file
+EXPOSE 7978 7979
```
Comment (clintonskitson) [inline Dockerfile]: Both should be equal to each other

**Comment by clintonskitson** [comment]
You're missing the gofig default configuration for setting these.

You should be able to run `polly env | grep HOST` and see it set to defaults even without a configuration.

File: Dockerfile (hunk 1 above)
Comment (dvonthenen) [inline Dockerfile]: I thought that was only on the command line with a -p
otherwise I can just divide on two separate lines.

**Comment by clintonskitson** [comment]
Search for `polly.host`

**Pull request closed without merging**